### DISCUSSION TITLE
Story flags

### DISCUSSION THREAD

**Discussion by MrsMarinaRose** [Meth962/SymphoniaSaveEdit#7] [General]
I know this is outside the scope of the editor you have, but it seems to me there must be a large portion of data saved that the editor doesn't go into, even in Manual Edit, such as the player location and which scenes have been viewed. Is there a way I could expose some of this data and tinker with it? I don't mind making my own fork of the editor to make this happen, I just want to do mad science on this game.

**Comment by MrsMarinaRose** [comment]
Oh, maybe it would simplify things if I mentioned I'm mainly interested in the GC version.

**Comment by Meth962** [comment]
Yes, there are tons of flags for every scene of the game. I started capturing some in notes long long ago for the GC but only did about an hour's worth of the game before going to the other world. It's a lot of work if you really want to do it.

**Reply by MrsMarinaRose** [reply]
Do you happen to know where the startpoint is for them with BaseStream.seek?
